packages/components/text/src/Text.tsx

```diff
@@ -15,10 +15,15 @@ import { textName, TextProps, TextTokens } from './Text.types';
 import { useTextTokens } from './TextTokens';
 import React from 'react';
 
+const emptyProps = {};
 export const Text = compressible<TextProps, TextTokens>((props: TextProps, useTokens: UseTokens<TextTokens>) => {
+  if (props === undefined) {
+    props = emptyProps;
+  }
+
   // split out color and variant from props
   const {
-    align,
+    align = undefined,
     block,
```

Comment:
It's not part of this PR, but looks like `block` prop is not used in Text.tsx

Reply:
Hmm I think that was something that was not fully resolved when Text was implemented? I'll let @lenahong decide what to do about that.